src/zeit/content/volume/browser/form.py

```diff
@@ -71,6 +87,42 @@ def setUpWidgets(self, *args, **kw):
         if not self.widgets['volume'].hasInput():
             self.widgets['volume'].setRenderedValue(settings.default_volume)
 
+    def add(self, object):
+        container = self.create_location(object)
+        if self._check_duplicate_volume(container):
+            return
+        container[VOLUME_FILENAME] = object
```

Comment:
1. Das passt so nicht mehr zu `retrieve_volume_using_info_from_metadata`.
2. Ich wäre dafür, den kompletten Namen in die products.xml reinzuschreiben (z.B. `location="http://xml.zeit.de/{year}/{name}/ausgabe"`), und nicht den VOLUME_FILENAME ausm Code zu holen. Damit können wir (1.) elegant lösen, und erhalten uns die Flexibilität, ggf. doch mal sowas wie `/zeit-magazin/ausgabe/2016/07` zu machen.